src/firejail/profile.c

```diff
@@ -1502,8 +1505,11 @@ int profile_check_line(char *ptr, int lineno, const char *fname) {
 			arg_rlimit_sigpending = 1;
 		}
 		else if (strncmp(ptr, "rlimit-as ", 10) == 0) {
-			check_unsigned(ptr + 10, "Error: invalid rlimit in profile file: ");
-			sscanf(ptr + 10, "%llu", &cfg.rlimit_as);
+			cfg.rlimit_as = parse_arg_size(ptr + 10);
+			if (cfg.rlimit_as == 0){
```

Comment:
if (cfg.rlimit_as == 0) {

(Fourth time's the charm I hope)

Reply:
🤦 is there any easy way to check code style before commit? 
this is exhausting for both of us.

Reply:
> facepalm is there any easy way to check code style before commit?

There is currently `make cppcheck`, though I don't think it does linting.

Either way, it's definitely been in my personal TODO for a while to:

1. Make 1 commit linting everything in the C codebase
2. Add a lint target to the Makefile and a lint check to the ci.

> this is exhausting for both of us.

Yes, very much so.  In the mean time, to help avoid that, please pay attention
to the exact changes suggested.  When not 100% sure, you could copy and paste
the suggestions into the code instead of editing by hand.